addons/project/static/src/views/project_task_kanban/project_task_kanban_renderer.js

```diff
@@ -33,7 +38,7 @@ export class ProjectTaskKanbanRenderer extends KanbanRenderer {
     }
 
     canCreateGroup() {
-        return (super.canCreateGroup() && this.isProjectTasksContext() && this.props.list.isGroupedByStage) || this.props.list.isGroupedByPersonalStages;
+        return (super.canCreateGroup() && this.isProjectTasksContext() && this.props.list.isGroupedByStage && (this.isProjectManager || !this.props.list.isGroupedByStage)) || this.props.list.isGroupedByPersonalStages;
```

Comment:
```suggestion
        return (super.canCreateGroup() && this.isProjectTasksContext() && this.props.list.isGroupedByStage && this.isProjectManager) || this.props.list.isGroupedByPersonalStages;
```
I don't think the second condition is useful (and besides, `A && (B || !A)` is equivalent to `A && B`)

Reply:
@MissingNoShiny Done
Thank You